includes/functions.php

```diff
@@ -550,18 +550,23 @@ function ifset( &$var, $return = false ) { // phpcs:ignore Universal.NamingConve
  * @return boolean if webmentions are open
  */
 function webmentions_open( $post = null ) {
-	$_post   = get_post( $post );
-	$post_id = $_post ? $_post->ID : 0;
-
-	// If the post type does not support Webmentions do not even check further
-	if ( ! post_type_supports( get_post_type( $post_id ), 'webmentions' ) ) {
-		return false;
+	$post    = get_post( $post );
+	$post_id = $post ? $post->ID : 0;
+	$open    = false;
+	if ( $post ) {
+		// Always consider the home mention link page to be open.
+		if ( get_option( 'webmention_home_mentions' ) === $post->ID ) {
+			$open = true;
+		} elseif ( ! post_type_supports( get_post_type( $post ), 'webmentions' ) ) {
+			// If the post type does not support Webmentions do not even check further.
+			$open = false;
+		} else {
+			// If the webmentions_closed meta ID has a value then consider webmentions to be open.
+			$open = get_post_meta( $post->ID, 'webmentions_closed', true );
+			$open = empty( $open ) ? true : false;
```

Comment:
```suggestion
			$closed = (int) get_post_meta( $post->ID, 'webmentions_closed','0' );
			$open   = ! (bool) $closed;
```

Reply:
Isn't WordPress always transforming booleans into integer-strings when using `get_post_meta`?

`true` => `'1'`
`false` => `'0'`

Reply:
How's the amendment?